### PR TITLE
TRANSLATE intermediate_source/speech_command_recognition_with_torchau…

### PR DESCRIPTION
**Pull request by yuseungwoo**
…dio_tutorial.py

## 라이선스 동의
_변경해주시는 내용에 BSD 3항 라이선스가 적용됨을 동의해주셔야 합니다._<br />
_더 자세한 내용은 [기여하기 문서](https://github.com/9bow/PyTorch-tutorials-kr/blob/master/CONTRIBUTING.md)를 참고해주세요._<br />
_동의하시면 아래 `[ ]`를 `[x]`로 만들어주세요._<br />

- [x] 기여하기 문서를 확인하였으며, 본 PR 내용에 BSD 3항 라이선스가 적용됨에 동의합니다.

## 관련 이슈 번호
_이 Pull Request와 관련있는 이슈 번호를 적어주세요._<br />
_이슈 또는 PR 번호 앞에 #을 붙이시면 제목을 바로 확인하실 수 있습니다. (예. #999 )_

- **이슈 번호**: #301(숫자)

## PR 종류
_이 PR에 해당되는 종류 앞의 `[ ]`을 `[x]`로 변경해주세요._<br />
- [ ] 오탈자를 수정하거나 번역을 개선하는 기여
- [x] 번역되지 않은 튜토리얼을 번역하는 기여
- [ ] 공식 튜토리얼 내용을 반영하는 기여
- [ ] 위 종류에 포함되지 않는 기여

## PR 설명
_이 PR로 무엇이 달라지는지 대략적으로 알려주세요._
#301 번역 완료

### CODE REVIEW

**Comment by yuseungwoo** [comment]
To @9bow @hyoyoung 

Please review my PR.

Appreciate your contribution. 

Sincerely 
Seung Woo

**Comment by hyoyoung** [comment]
pr종류 선택이 빠졌습니다

File: intermediate_source/speech_command_recognition_with_torchaudio_tutorial.py [hunk 1]
```diff
@@ -251,20 +233,16 @@ def collate_fn(batch):
...
-# around 10ms (and at 4kHz, around 20 ms). This size is similar to speech
-# processing applications that often use receptive fields ranging from
-# 20ms to 40ms.
+# 본 튜토리얼에선 합성곱 신경망을 사용하여 오디오 데이터를 처리합니다.
+# 보통은 고급 변형 기법을 적용하여 오디오 데이터를 처리하지만, 합성곱 신경망은 오디오를 가공치 않고
+# 정확하게 처리하는데 사용합니다. 자세한 신경망 구조는 M5 신경망 구조 이후에 설계되었고 `이 논문 <https://arxiv.org/pdf/1610.00087.pdf>`__ 에 자세한 설명을 해 놓았습니다.
+# 오디오를 가공치 않고 처리하는 모델의 중요한 측면은 첫번째 층 필터들의 수용 영역입니다.
+# 설계한 모델의 첫번째 길이는 80입니다. 그래서 8kHz으로 샘플링한 오디오 데이터를 처리할 때, 
+# 수용 영역은 대략 10ms 입니다. (샘플링이 4kHz일 경우, 대략 20ms). 이 사이즈는
+# 20ms 에서 40ms 길이의 수용 영역을 갖는 음성 어플리케이션과 유사합니다.
```
Comment (puhuk) [inline intermediate_source/speech_command_recognition_with_torchaudio_tutorial.py:245]: "음성 어플리케이션" -> "발화 처리 어플리케이션" 또는 "음성 처리 어플리케이션" 이 어떨지 의견드립니다.

**Review by puhuk — COMMENTED** [review]
고생하셨습니다.
몇 가지 리뷰 코멘트 드립니다. :)

File: intermediate_source/speech_command_recognition_with_torchaudio_tutorial.py [hunk 2]
```diff
@@ -1,22 +1,18 @@
...
-“Change runtime type”. In the pop-up that follows, you can choose GPU.
-After the change, your runtime should automatically restart (which means
-information from executed cells disappear).
+Colab엔 GPU 옵션이 있습니다. 메뉴탭에 있는 “런타임” 에서 “런타임 유형 변경” 을 선택합니다. 
+팝업창에서, GPU를 선택합니다. 변경 후, 런타임은 자동적으로 재시작합니다.(실행하고 있던 셀의 정보는 사라지게 됩니다)
 
-First, let’s import the common torch packages such as
-`torchaudio <https://github.com/pytorch/audio>`__ that can be installed
-by following the instructions on the website.
+먼저, 웹사이트 공지를 통해 설치한 `torchaudio <https://github.com/pytorch/audio>`__ 와 같은 공통으로 사용하고 있는 torch 패키지를 로드합니다.
```
Comment (des00) [inline intermediate_source/speech_command_recognition_with_torchaudio_tutorial.py:11]: 줄바꿈이 있으면 어떨까 싶습니다.

File: intermediate_source/speech_command_recognition_with_torchaudio_tutorial.py [hunk 3]
```diff
@@ -38,34 +34,28 @@
...
+# 데이터셋 가져오기
 # ---------------------
 #
-# We use torchaudio to download and represent the dataset. Here we use
-# `SpeechCommands <https://arxiv.org/abs/1804.03209>`__, which is a
-# datasets of 35 commands spoken by different people. The dataset
-# ``SPEECHCOMMANDS`` is a ``torch.utils.data.Dataset`` version of the
-# dataset. In this dataset, all audio files are about 1 second long (and
-# so about 16000 time frames long).
+# 데이터셋을 다운받고 보기 위해 torchaudio를 사용합니다.
```
Comment (des00) [inline intermediate_source/speech_command_recognition_with_torchaudio_tutorial.py:49]: "보기 위해"도 괜찮은 표현 같습니다만 "살펴보기 위해" 혹은 "보여주기 위해" 라는 표현은 어떠신지요.

File: intermediate_source/speech_command_recognition_with_torchaudio_tutorial.py [hunk 4]
```diff
@@ -111,16 +100,16 @@ def load_list(filename):
 
 
 ######################################################################
-# Let’s find the list of labels available in the dataset.
+# 데이터셋에서 사용할 발화를 확인해 봅시다.
```
Comment (spongebob03) [inline intermediate_source/speech_command_recognition_with_torchaudio_tutorial.py:103]: 음성 인식에 관한 튜토리얼이기 때문에 발화가 예측하고자 하는 대상인 것은 확실하지만 label을 그대로 `라벨`이라고 표현한다면 예측 대상임을 좀 더 보편적으로 인식할 수 있다고 봅니다.  (아직 용어집에 추가되지 않았지만 [이슈](https://github.com/9bow/PyTorch-tutorials-kr/issues/297)가 올라와 있어 용어 고정 후 반영하면 좋을거 같습니다.)

File: intermediate_source/speech_command_recognition_with_torchaudio_tutorial.py [hunk 5]
```diff
@@ -91,17 +81,16 @@ def load_list(filename):
...
 
 waveform, sample_rate, label, speaker_id, utterance_number = train_set[0]
 
 
 ######################################################################
-# A data point in the SPEECHCOMMANDS dataset is a tuple made of a waveform
-# (the audio signal), the sample rate, the utterance (label), the ID of
-# the speaker, the number of the utterance.
+# SPEECHCOMMANDS 데이터셋에서 데이터 포인트는 wav(음성 신호), 
+# 샘플링 주파수, 발화(텍스트), 화자의 ID, 발화 개수로 구성한 튜플입니다.
```
Comment (spongebob03) [inline intermediate_source/speech_command_recognition_with_torchaudio_tutorial.py:93]: 발화(label)의 label은 텍스트보다는 예측 대상 값을 의미하는 `라벨`이라고 보여집니다. 
아래 코맨트와 마찬가지로 그대로 `라벨` 표기하는 것은 어떤가요..?

File: intermediate_source/speech_command_recognition_with_torchaudio_tutorial.py [hunk 6]
```diff
@@ -184,38 +170,34 @@ def index_to_label(index):
...
-# into two batched tensors for the model, we implement a collate function
-# which is used by the PyTorch DataLoader that allows us to iterate over a
-# dataset by batches. Please see `the
-# documentation <https://pytorch.org/docs/stable/data.html#working-with-collate-fn>`__
-# for more information about working with a collate function.
-#
-# In the collate function, we also apply the resampling, and the text
-# encoding.
+# 오디오 데이터와 발화로 구성하고 있는 데이터 포인트를 모델에 적용하기 위한 두 배치로 반환하기 위해 
+# 배치를 반복해서 처리하는 PyTorch DataLoader 에서 사용하는 collate 함수를 구현합니다
```
Comment (Taeyoung96) [inline intermediate_source/speech_command_recognition_with_torchaudio_tutorial.py:174]: '사용하는 collate 함수를 구현합니다' 뒤에 온점이 있으면 좋을 것 같습니다.  
:point_right: `사용하는 collate 함수를 구현합니다.`

File: intermediate_source/speech_command_recognition_with_torchaudio_tutorial.py (hunk 1 above)
Comment (Taeyoung96) [reply intermediate_source/speech_command_recognition_with_torchaudio_tutorial.py:245]: 맞춤법상, '해 놓았습니다.'를 '해놓았습니다.'로 바꾸는 것이 맞는 것 같습니다.

File: intermediate_source/speech_command_recognition_with_torchaudio_tutorial.py [hunk 7]
```diff
@@ -318,25 +296,21 @@ def count_parameters(model):
...
-# Training and Testing the Network
+# 신경망 학습 및 테스트
 # --------------------------------
 #
-# Now let’s define a training function that will feed our training data
-# into the model and perform the backward pass and optimization steps. For
-# training, the loss we will use is the negative log-likelihood. The
-# network will then be tested after each epoch to see how the accuracy
-# varies during the training.
+# 학습 데이터를 모델에 적용하고 역전파 및 최적화를 수행하는 학습 함수를 정의 합니다.
```
Comment (Taeyoung96) [inline intermediate_source/speech_command_recognition_with_torchaudio_tutorial.py:311]: 맞춤법상 '정의 합니다.'를 '정의합니다.'로 바꿔야 할 것 같습니다!

File: intermediate_source/speech_command_recognition_with_torchaudio_tutorial.py [hunk 8]
```diff
@@ -318,25 +296,21 @@ def count_parameters(model):
...
+# 신경망 학습 및 테스트
 # --------------------------------
 #
-# Now let’s define a training function that will feed our training data
-# into the model and perform the backward pass and optimization steps. For
-# training, the loss we will use is the negative log-likelihood. The
-# network will then be tested after each epoch to see how the accuracy
-# varies during the training.
+# 학습 데이터를 모델에 적용하고 역전파 및 최적화를 수행하는 학습 함수를 정의 합니다. 
+# 학습에서, 음수 로그 가능도 함수를 손실로 활용을 합니다.
```
Comment (Taeyoung96) [inline intermediate_source/speech_command_recognition_with_torchaudio_tutorial.py:312]: 앞뒤 문맥에서 사용했던 어미에 통일성을 주기위해, '활용을 합니다.'를 '활용합니다.'로 바꾸는 것은 어떨까요?

File: intermediate_source/speech_command_recognition_with_torchaudio_tutorial.py [hunk 9]
```diff
@@ -476,9 +447,9 @@ def predict(tensor):
 
 
 ######################################################################
-# Feel free to try with one of your own recordings of one of the labels!
-# For example, using Colab, say “Go” while executing the cell below. This
-# will record one second of audio and try to classify it.
+# 발화중 하나를 자유롭게 녹음하십시오!
```
Comment (Taeyoung96) [inline intermediate_source/speech_command_recognition_with_torchaudio_tutorial.py:450]: 맞춤법상 '발화중'을 '발화 중'으로 쓰는 것이 적절해 보입니다.

File: intermediate_source/speech_command_recognition_with_torchaudio_tutorial.py [hunk 10]
```diff
@@ -131,26 +120,23 @@ def load_list(filename):
...
-#
-# We don’t need to apply other transformations here. It is common for some
-# datasets though to have to reduce the number of channels (say from
-# stereo to mono) by either taking the mean along the channel dimension,
-# or simply keeping only one of the channels. Since SpeechCommands uses a
-# single channel for audio, this is not needed here.
+# 데이터 셋 구성을 마친 후, 데이터 변형을 적용합니다. 오디오 분류 특성을 줄이지 않는 선에서 
+# 보다 빠른 처리를 위해 오디오를 다운 샘플링 합니다.
+
+# 여기서 다른 데이터 변형은 필요하지 않습니다. 오디오 채널 수를 줄이기 위하여(스테테오 에서 모노)
```
Comment (joonkeekim) [inline intermediate_source/speech_command_recognition_with_torchaudio_tutorial.py:137]: 스테테오 -> 스테레오로 고쳐야할 것 같습니다!

File: intermediate_source/speech_command_recognition_with_torchaudio_tutorial.py [hunk 11]
```diff
@@ -396,24 +369,23 @@ def test(model, epoch):
...
 
     print(f"\nTest Epoch: {epoch}\tAccuracy: {correct}/{len(test_loader.dataset)} ({100. * correct / len(test_loader.dataset):.0f}%)\n")
 
 
 ######################################################################
-# Finally, we can train and test the network. We will train the network
-# for ten epochs then reduce the learn rate and train for ten more epochs.
-# The network will be tested after each epoch to see how the accuracy
-# varies during the training.
+# 마지막으로, 신경망을 학습하고 테스트 할 수 있게 되었습니다.
```
Comment (joonkeekim) [inline intermediate_source/speech_command_recognition_with_torchaudio_tutorial.py:386]: Finally는 마지막으로 보단 마침내로 번역하는게 어떨까요??

File: intermediate_source/speech_command_recognition_with_torchaudio_tutorial.py [hunk 12]
```diff
@@ -525,21 +496,19 @@ def record(seconds=1):
...
 #
-# In this tutorial, we used torchaudio to load a dataset and resample the
-# signal. We have then defined a neural network that we trained to
-# recognize a given command. There are also other data preprocessing
-# methods, such as finding the mel frequency cepstral coefficients (MFCC),
-# that can reduce the size of the dataset. This transform is also
-# available in torchaudio as ``torchaudio.transforms.MFCC``.
+# 본 튜토리얼에선, 데이터셋을 로드하고 신호를 재조정 하기위해 torchaudio를 사용했습니다. 
+# 주어진 명령어를 인식하게끔 학습한 신경망을 정의하였습니다.
+# 또다른 데이터 전처리 방법도 존재합니다. 가령 mel frequency cepstral coefficients (MFCC)를 찾을 수 있고,
```
Comment (joonkeekim) [inline intermediate_source/speech_command_recognition_with_torchaudio_tutorial.py:512]: 또다른 -> 또 다른 으로 띄워야 할 것 같습니다!

**Review by joonkeekim — COMMENTED** [review]
몇 가지 코멘트 남겼습니다!

File: intermediate_source/speech_command_recognition_with_torchaudio_tutorial.py (hunk 3 above)
Comment (YonghyunRyu) [reply intermediate_source/speech_command_recognition_with_torchaudio_tutorial.py:49]: 보통 딥러닝 분야에서 represent 를 보다로 해석하지는 않는데요.
알아보고 해야할 것 같습니다.

File: intermediate_source/speech_command_recognition_with_torchaudio_tutorial.py (hunk 5 above)
Comment (YonghyunRyu) [reply intermediate_source/speech_command_recognition_with_torchaudio_tutorial.py:93]: label 은 레이블이 맞지 않을까요. 저는 관련 연구하면서 라벨이라고 말하는 경우를 들어보질 못했네요.

File: intermediate_source/speech_command_recognition_with_torchaudio_tutorial.py [hunk 13]
```diff
@@ -131,26 +120,23 @@ def load_list(filename):
...
-# This is a good place to apply transformations to the data. For the
-# waveform, we downsample the audio for faster processing without losing
-# too much of the classification power.
-#
-# We don’t need to apply other transformations here. It is common for some
-# datasets though to have to reduce the number of channels (say from
-# stereo to mono) by either taking the mean along the channel dimension,
-# or simply keeping only one of the channels. Since SpeechCommands uses a
-# single channel for audio, this is not needed here.
+# 데이터 셋 구성을 마친 후, 데이터 변형을 적용합니다. 오디오 분류 특성을 줄이지 않는 선에서
```
Comment (YonghyunRyu) [inline intermediate_source/speech_command_recognition_with_torchaudio_tutorial.py:134]: 번역하면서 원문의 뜻이 바뀌는 것 같아요.

File: intermediate_source/speech_command_recognition_with_torchaudio_tutorial.py (hunk 3 above)
Comment (des00) [reply intermediate_source/speech_command_recognition_with_torchaudio_tutorial.py:49]: 데이터를 표현한다는 것도 누군가가 봐 주기 바라기 때문에 보기 위해 라고 번역하신 듯 합니다.
딥러닝 분야에서 사용하는 데이터 표현형이나 표현 방식을 나타내는 data representation이 아니라서
보다 라는 동사의 파생 의미로 사용해도 별 상관은 없다고 생각합니다.

**Comment by hyoyoung** [comment]
feedback이 없어서 pr을 close합니다.
필요할때 열어주시거나 새로 생성하여주세요.
